tiledb/sm/query/query.cc

```diff
@@ -695,10 +696,12 @@ Status Query::init() {
       return LOG_STATUS(Status::QueryError(errmsg.str()));
     }
 
+    auto* initialization_subarray_if_any =
+        have_initialization_subarray_ ? &initialization_subarray_ : nullptr;
     if (type_ == QueryType::READ) {
-      RETURN_NOT_OK(reader_.init(layout_));
+      RETURN_NOT_OK(reader_.init(layout_, initialization_subarray_if_any));
     } else {  // Write
-      RETURN_NOT_OK(writer_.init(layout_));
+      RETURN_NOT_OK(writer_.init(layout_, initialization_subarray_if_any));
```

Comment:
We don't initialize the reader/writer with ranges today, why do we need this?

Reply:
removing
I suspect a train of thought involving when init()s occurred vs possible setting of the subarray and its being .is_set() or not led to that,
but currently finding no past/present active references to its use anywhere in any history of any branches of my working repository, whatever the thought was, it is abandoned.